Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/transformation/TransformationManager.java

```diff
@@ -22,18 +25,26 @@
 
     @Autowired
     private ITransformFilesDAO transformationDAO;
+    
+    @Autowired
+    private ITransformationSaveService transformationFileService;
 
     @Transactional
     @Override
-    public void saveTransformation(String title, String description,
-            String patternFileName, String patternTitle,
-            String patternDescription, String mappingFileName,
-            String mappingTitle, String mappingDescription,String userName) {
-        TransformFilesDTO tranformDTO = new TransformFilesDTO(title,
-                description, patternFileName, patternTitle, patternDescription,
-                mappingFileName, mappingTitle, mappingDescription, userName);
+    public void saveTransformations(ITransformationFile transformationFile){
+        TransformFilesDTO tranformDTO = new TransformFilesDTO(transformationFile.getTitle(),
+                transformationFile.getDescription(), transformationFile.getPatternFileName(), transformationFile.getPatternTitle(), 
+                transformationFile.getPatternDescription(), transformationFile.getMappingFileName(), transformationFile.getMappingTitle(), 
+                transformationFile.getMappingDescription(), transformationFile.getUserName());
         tranformDTO.setId(transformationDAO.generateUniqueID());
         transformationDAO.saveNewDTO(tranformDTO);
+        
+        try{
+        transformationFileService.saveFileToLocal(transformationFile);
```

Comment:
@mkarthik90, yes, that would be really helpful! @yyk16293, but please make also sure that your other formatting settings are correct in Eclipse. It should have indented that line for you when you hit enter to create a new line.